src/java/org/apache/cassandra/repair/consistent/SyncStatSummary.java

```diff
@@ -147,7 +152,22 @@ public String toString()
             }
             StringBuilder output = new StringBuilder();
 
-            output.append(String.format("%s.%s - %s ranges, %s sstables, %s bytes\n", keyspace, table, ranges, files, FBUtilities.prettyPrintMemory(bytes)));
+            output.append(String.format("%s.%s - %s ranges, %s sstables, %s bytes\n", keyspace, table, ranges.size(), files, FBUtilities.prettyPrintMemory(bytes)));
+            if (ranges.size() > 0)
+            {
+                output.append("    Mismatching ranges: ");
+                int i = 0;
+                Iterator<Range<Token>> rangeIterator = ranges.iterator();
+                while (rangeIterator.hasNext() && i < 30)
```

Comment:
This seems to be like arbitrary number - "30". Is there some intrinsic logic behind that figure or was it chosen empirically? Anyway, I do not think we need to make this configurable, maybe extraction to a constant would be nice.